### PR TITLE
Front  merge into Main

### PR DESCRIPTION
**Pull request by trevor256**

### CODE REVIEW

**Comment by trevor256** [comment]
I'm getting a DateTimeParseException on the back-end that wont let the tasks render, other than that everything else is good. 

Caused by: java.time.format.DateTimeParseException: Text '' could not be parsed at index 0
at java.base/java.time.format.DateTimeFormatter.parseResolved0(DateTimeFormatter.java:2052) ~[na:na]
at java.base/java.time.format.DateTimeFormatter.parse(DateTimeFormatter.java:1954) ~[na:na]
at java.base/java.time.Instant.parse(Instant.java:397) ~[na:na]
at org.springframework.data.convert.Jsr310Converters$StringToInstantConverter.convert(Jsr310Converters.java:321) ~[spring-data-commons-3.2.2.jar:3.2.2]
at org.springframework.data.convert.Jsr310Converters$StringToInstantConverter.convert(Jsr310Converters.java:313) ~[spring-data-commons-3.2.2.jar:3.2.2]
at org.springframework.core.convert.support.GenericConversionService$ConverterAdapter.convert(GenericConversionService.java:358) ~[spring-core-6

**Review by TobyMarch — DISMISSED** [review]

File: web-app/src/Home.js [hunk 1]
```diff
@@ -183,10 +192,11 @@ useEffect(() => {
         owner,
         title,
         description,
-        startDate: new Date(startDate).toISOString(),
```
Comment (TobyMarch) [inline web-app/src/Home.js:186]: We need this conversion to send the date in the format that the back-end is expecting

File: web-app/src/Home.js [hunk 2]
```diff
@@ -183,10 +192,11 @@ useEffect(() => {
...
-        dueDate: new Date(dueDate).toISOString(),
+        createdDate,
+        startDate,
+        dueDate,
         priority,
-        duration,
+         duration,
```
Comment (TobyMarch) [inline web-app/src/Home.js:199]: And this is just cosmetic, but we might as well even up the spacing

File: web-app/src/Home.js [hunk 3]
```diff
@@ -136,27 +163,9 @@ const priorityGradientStyles = [
     return newTaskFail;
   };
 
-const fetchArchivedTasks = async () => {
```
Comment (TobyMarch) [inline web-app/src/Home.js:139]: Are you planning to re-add the archived tasks logic somewhere else?

**Pull request merged**